article/_posts/2020-11-19-AlphaMuGoZero.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: review
+title: "AlphaGo/AlphaGoZero/AlphaZero/MuZero: Mastering games using progressively fewer priors"
+tags: deep-learning reinforcement
+author: "Antoine Théberge"
+cite:
+    authors: "Julian Schrittwieser, Ioannis Antonoglou, Thomas Hubert, Karen Simonyan, Laurent Sifre, Simon Schmitt, Arthur Guez, Edward Lockhart, Demis Hassabis, Thore Graepel, Timothy Lillicrap, David Silver"
+    title:   "Mastering Atari, Go, Chess and Shogi by Planning with a Learned Model"
+    venue:   "arXiv"
+pdf: ""
```

Comment:
Could you please add the link to the paper?

Reply:
Done in 137ef52 !